src/wombats/daos/game.clj

```diff
@@ -105,7 +112,7 @@
     (let [frame-tmp-id (d/tempid :db.part/user)
           frame-trx {:db/id frame-tmp-id
                      :frame/frame-number 0
-                     :frame/round-number 0
+                     :frame/round-number 1
```

Comment:
I think it would be good to eventually figure out why round-number of 0 doesn't work. Would rather there not be an inconsistency between round and frame indexing.